lib/src/value/calculation.dart

```diff
@@ -120,6 +122,151 @@ class SassCalculation extends Value {
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value atan(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("atan", [argument]);
+    }
+    return number_lib.atan(argument);
+  }
+
+  /// Creates a `asin()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value asin(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("asin", [argument]);
+    }
+    return number_lib.asin(argument);
+  }
+
+  /// Creates a `acos()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value acos(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber || argument.hasUnit('%')) {
+      return SassCalculation._("acos", [argument]);
+    }
+    return number_lib.acos(argument);
+  }
+
+  /// Creates a `abs()` calculation with the given [argument].
+  ///
+  /// The [argument] must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value abs(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber) {
+      return SassCalculation._("abs", [argument]);
+    }
+    if (argument.hasUnit("%")) {
+      warnForDeprecation(
+          "abs() will not longer support percentage units."
+          "future release.\n"
+          "\n"
+          "More info: https://sass-lang.com/documentation/values/numbers",
+          Deprecation.absPercent);
+      return SassCalculation._("abs", [argument]);
```

Comment:
This case is for backwards compatibility with the current global Sass function, so the current behavior should just be to call `abs` the same as we do for an argument with known units (returning a calculation here will only happen _after_ the deprecation process is complete.

Reply:
Got it, thanks!